README.md

```diff
@@ -31,28 +31,28 @@ the form of OCI container images with [apko][apko].
 You can install Melange from Homebrew:
 
 ```shell
-brew install melange
+$ brew install melange
 ```
 
 You can also install Melange from source:
 
 ```shell
-go install chainguard.dev/melange@latest
+$ go install chainguard.dev/melange@latest
 ```
 
 You can also use the Melange container image:
 
 ```shell
-docker run cgr.dev/chainguard/melange version
+$ docker run cgr.dev/chainguard/melange version
 ```
 
 To use the examples, you'll generally want to mount your current directory into the container and provide elevated privileges, e.g.:
 
 ```shell
-docker run --privileged -v "$PWD":/work cgr.dev/chainguard/melange build examples/gnu-hello.yaml
+$ docker run --privileged -v "$PWD":/work cgr.dev/chainguard/melange build examples/gnu-hello.yaml
 ```
 
-Running outside of a container requires [Docker](https://docs.docker.com/get-docker/), but should also work with other runtimes such as [podman](https://podman.io/getting-started/installation).
+Running outside of a container requires [Docker](https://docs.docker.com/get-docker/), but should also work with other run-times such as [podman](https://podman.io/getting-started/installation).
```

Comment:
Runtime is correct here, Run-time is not.  https://en.wikipedia.org/wiki/Runtime_system